### ISSUE TITLE
[FEATURE] Show dependent `.dll`s

### ISSUE THREAD

**Issue by Enough7** [madushans/DependenSee#9]
**Is your feature request related to a problem?**
No

**Describe the solution you'd like**
I have some `.dll`s in a seperate project in my solution. When I reference these `.dll`s in my `.csproj` filde dependensee should show them in the dependency graph.

**Describe alternatives you've considered**
None

**Additional context**
For example [Zenject](https://github.com/modesttree/Zenject) does not offer a nuget-package so one has to download Zenject as `.dll`s and reference them in you project themself. DependenSee is not able to detect such a dependency. It looks like this in my `.csproj`-file:
```
...
<Reference Include="Zenject, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
        <HintPath>..\Zenject\Zenject.dll</HintPath>
</Reference>
...
```

**Comment by madushans** [comment]
Yep that looks about right. Will work on adding this to the next release. 
Unfortunately no timeline yet.

Will update here when ready.

**Comment by madushans** [comment]
This is now fixed in [1.2.2](https://github.com/madushans/DependenSee/releases/tag/1.2.2)

Please run `dotnet tool update DependenSee --global` to update.
Note that with this update, .NET Framework projects will start showing referenced BCL DLLs (i.e System, Microsoft.CSharp) as well.

Thanks for using DependenSee ♥

**Comment by Enough7** [comment]
Can not wait to try it out, thank you
